web_notify/__manifest__.py

```diff
@@ -6,9 +6,11 @@
     'name': 'Web Notify',
     'summary': """
         Send notification messages to user""",
-    'version': '9.0.1.0.0',
+    'version': '10.0.1.0.0',
+    'description': 'Web Notify',
     'license': 'AGPL-3',
-    'author': 'ACSONE SA/NV,Odoo Community Association (OCA)',
+    'author': """Serpent Consulting Services Pvt. Ltd.,
```

Comment:
@JayVora-SerpentCS  Thank you for the migration but you are not the the author of this addons. IMO a change in the author can be done when you contribute to the module with a new functionality or a refactoring of the code not for a simple migration. Moreover IMO a find and replace of `openerp`  by `odoo` doesn't justify a new entry in the contributor list.